nmrcraft/analysis/plotting.py

```diff
@@ -84,29 +84,60 @@ def plot_predicted_vs_ground_truth_density(
     plt.show()
 
 
-def plot_confusion_matrix(cm, classes, title, path):
+def plot_confusion_matrix(
+    cm, classes, title, path, full=True, columns_set=False
+):
     """
     Plots the confusion matrix.
     Parameters:
     - cm (array-like): Confusion matrix data.
     - classes (list): List of classes for the axis labels.
     - title (str): Title of the plot.
+    - full (bool): If true plots one big, else many smaller.
```

Comment:
I like the description hehe

Reply:
Alternatively for the final review round in the end, we could rename to “Plots single confusion matrix for all targets combined vs one CM per target” or something like that